test/uk/gov/hmrc/alcoholdutyreturns/service/AuditServiceSpec.scala

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
Might be beneficial to add integration tests as well

Reply:
I've added to the existing tests. Not sure if added a check that the audit code calls out to datastream is worth it in integration tests (unless code is provided) since any mocking of the return would be broken if the response changes as the connector is out of our control). There are tests that the audit service gets called.